easybuild/scripts/rpath_wrapper_template.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Should we not be smarter here and use the fact that when we know when we need to use the prefix bash? I can see a scenario where the prefix bash is not in the path but that is one we should expect to use.

Reply:
I know that this will work with the current EESSI setup, but I'm thinking of endusers who won't drop into a prefix shell to install software

Reply:
Are you saying we should should leave this to `/bin/bash` and instead fiddle with the shebang line when we use the template to create the actual shell wrappers, when EasyBuild is configured with `--sysroot`?

Reply:
Using `<sysroot>/bin/bash` may actually be a bad idea, since if we're not in a Prefix environment we would be relying on libraries provided by the host to run the `bash` from the EESSI compat layer, that would potentially lead to trouble...

Reply:
Well, you are right in general, there is no guarantee that the prefix bash will actually work on the build system anyway